roles/confluent.variables/defaults/main.yml

```diff
@@ -133,10 +133,10 @@ ssl_custom_certs_remote_src: false
 
 # Zookeeper Variables
 
-### Only use to customize Linux User Zookeeper Service runs with. User must exist on host.
+### Only use to customize Linux User Zookeeper Service runs with. Default user is cp-kafka
```

Comment:
Modify variable to customize the Linux User that the Zookeeper Service runs with. Default user is cp-kafka.

Reply:
How about "Set this variable..." instead of "Modify..." ?

Modify makes me think they already have it set, which they should not.